src/base/orgs/View/Projects.svelte

```diff
@@ -67,17 +75,19 @@
       {@const pendingAnchor = pendingAnchors[project.urn]}
       {#if project.head}
         <div class="project">
-          <Widget {project} {anchor} on:click={() => onClick(project)}>
-            <span class="actions" slot="actions">
-              {#if profile?.org?.safe && account && anchor}
-                {#if pendingAnchor} <!-- Pending anchor -->
-                  <AnchorActions
-                    {account} {config} anchor={pendingAnchor} safe={profile.org.safe}
-                    on:success={() => loadAnchors()} />
+          {#await Project.getCommits(project.urn, seed.api, project.head, getTimestampOneYearAgo()) then points}
```

Comment:
I think we discussed this, but I'm not too happy with waiting for the commit data to load before we are able to show the widget. Can this be moved so that the widget renders without waiting?